app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -217,6 +217,8 @@ def async_state_done(current_async_state)
         idv_session.mark_verify_info_step_complete!
         idv_session.invalidate_steps_after_verify_info!
 
+        resolution_rate_limiter.increment!
```

Comment:
I think we need to continue incrementing in `#async_state_done`. A few lines above we make this a no-op if the user submits with a job in progress. That _should_ give us some protection against double submissions on the update action.

I think we need to modify `#summarize_result_and_rate_limit_failures` to be `#summarize_result_and_rate_limit` and call `resolution_rate_limiter.increment!` there instead of in `#idv_failure`.

Additionally, we don't want to increment in the case of an exception still. The current implementation looks like this:

```ruby
resolution_rate_limiter.increment! if proofing_results_exception.blank?
```

We want to keep that around, but also call it in the success context. Moving it up the stack into `#summarize_result_and_rate_limit_failures` from `#idv_failure` should do that for us.

Reply:
Got it - done in c9483de90

Reply:
Done in c9483de